spec/support/example_app_helper.rb

```diff
@@ -0,0 +1,37 @@
+require 'fileutils'
+
+module ExampleAppHelper
+  def create_example_app
+    FileUtils.cd(tmp_path) do
+      `rails new #{app_name} -d postgresql --skip-action-mailer --skip-action-mailbox --skip-action-cable --skip-sprockets --skip-listen --skip-javascript --skip-turbolinks --skip-system-test --skip-test-unit --skip-bootsnap --skip-spring`
+    end
+  end
+
+  def remove_example_app
+    FileUtils.rm_rf(example_app_path)
+  end
+
+  def install_good_job
+    File.open("#{example_app_path}/Gemfile", 'a') do |f|
+      f.puts "gem 'good_job'"
+    end
```

Comment:
I think this should be moved into the `create_example_app` because I think it will probably always be used for set up.

Reply:
Sure. What should we call the method as it's no longer just creating an app? 🤔

Reply:
Good question. I don't think "setup_example_app" collides with anything.